apps/web/views.py

```diff
@@ -60,10 +60,19 @@ def contact_us(request):
 
 
 @throttle_classes([AnonRateThrottle])
-@api_view(['GET'])
+@api_view(['GET', 'POST'])
 @permission_classes((permissions.AllowAny,))
 def our_team(request):
-    teams = Team.objects.all()
-    serializer = TeamSerializer(teams, many=True, context={'request': request})
-    response_data = serializer.data
-    return Response(response_data, status=status.HTTP_200_OK)
+    if request.method == 'GET':
+        teams = Team.objects.all()
+        serializer = TeamSerializer(teams, many=True, context={'request': request})
+        response_data = serializer.data
+        return Response(response_data, status=status.HTTP_200_OK)
+    elif request.method == 'POST':
+        request.data['team_type'] = Team.CONTRIBUTOR
+        serializer = TeamSerializer(data=request.data)
+        if serializer.is_valid():
+            serializer.save()
+            response_data = {'message', 'The contributor has been added.'}
```

Comment:
Please replace `{'message', 'The contributor has been added.'}` with `{'message': 'Successfully added the contributor'}`